### ISSUE TITLE
Set up development environment to be more flexible

### ISSUE THREAD

**Issue by jleung51** [jleung51-coursework/phaser#22]
Currently, the default and UnitTest++ directories in `build/CMakeLists.txt` are set thusly:

```
set(USER_DIR /home/vagrant)
...
set(Test_DIR ${USER_DIR}/cmpt276-projects/unittest-cpp)
```

However, this assumes the user is running a Vagrant virtual machine and has set up their development environment exactly as we have set up ours.

I want to make this more flexible, so that nobody is restricted to a virtual machine.

**Comment by jleung51** [comment]
Working on this in the branch `dev-environment`!

**Comment by daphne-chong** [comment]
Implemented in pull request #23
